script/generate

```diff
@@ -0,0 +1,243 @@
+#  IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+#  ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+#  LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+#  CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+#  SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+#  INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+#  CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+#  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+#  POSSIBILITY OF SUCH DAMAGE.
+#
+
+set -euo pipefail
+
+# ==============================================================================
+# Bash definitions
+if [[ -n ${BASH_SOURCE[0]} ]]; then
+    script_path="${BASH_SOURCE[0]}"
+else
+    script_path="$0"
+fi
+
+script_dir="$(dirname "${script_path}")"
+repo_dir="$(git rev-parse --show-toplevel)"
+sdk_dir="${repo_dir}/third_party/silabs/gecko_sdk"
+
+# shellcheck source=script/slc_cli
+source "${repo_dir}/script/slc_cli"
+
+# shellcheck source=script/util
+source "${repo_dir}/script/util"
+
+# ==============================================================================
+trust_sdk_and_extensions()
+{
+    # TODO: Figure out a way to skip the trust command if the SDK or extensions are already trusted
+
+    # List of dirs containing SDK extensions to trust
+    # NOTE: this list must only contain absolute paths since it will be used to generate symlinks
+    extension_dirs=(
+        # ot-efr32 extension
+        "${repo_dir}"
+    )
+
+    # Add a vendor slc extension if the path to one is defined
+    if [ -n "${VENDOR_EXTENSION-}" ]; then
+        local vendor_extension_abs_path=""
+        case "${VENDOR_EXTENSION}" in
+            /*)
+                # absolute path
+                vendor_extension_abs_path="${VENDOR_EXTENSION}"
+                ;;
+            *)
+                # relative path
+                vendor_extension_abs_path=$(realpath "$(pwd)/${VENDOR_EXTENSION}")
+                if [ ! -d "${vendor_extension_abs_path}" ]; then
+                    set +x
+                    echo "VENDOR_EXTENSION=${VENDOR_EXTENSION}"
+                    echo "script_dir=${script_dir}"
+                    echo "vendor_extension_abs_path=${vendor_extension_abs_path}"
+                    echo "ERROR: '${vendor_extension_abs_path}' does not exist"
+                    exit 5
+                fi
+                ;;
+        esac
+
+        extension_dirs+=("${vendor_extension_abs_path}")
+    fi
+
+    set +x
+    echo "======================================================================"
+    echo "Trusting Gecko SDK              :'${sdk_dir}':"
+    echo "======================================================================"
+    set -x
+
+    # Trust the Gecko SDK submodule
+    run_slc -v 1 signature trust --sdk "${sdk_dir}" -data "${openthread_slc_data}"
```

Comment:
Curious, what is `openthread_slc_data`?

Reply:
The `-data` flag moves some internal data into a different location from the default. This location is where the trust storage is located by default. In environments where it might be possible that multiple SLC processes might be running concurrently, this helps prevent the SLC processes from conflicting with each other.